pkg/networkverifier/networkverifier.go

```diff
@@ -56,13 +57,14 @@ func InitializeValidateEgressInput(cluster *v1.Cluster, clusterDeployment *hivev
 		"Name":                 "osd-network-verifier",
 	}
 
-	region := cluster.Region().ID()
-	if onvAwsClient.GetAMIForRegion(region) == "" {
-		return nil, fmt.Errorf("unsupported region: %s", region)
+	// We default to using the `curl` based probe for validating egress at this time
+	cloudImageID, err := curl.Probe{}.GetMachineImageID(cloud.AWSClassic, cpu.ArchX86, cluster.Region().ID())
+	if err != nil {
+		return nil, fmt.Errorf("failed getting machine image ID: %w", err)
```

Comment:
No need to even set cloudImageID; the verifier will pick a sane default (using effectively the same logic you articulate here) when the `CloudImageID` field is left unset